c/indexer.c

```diff
@@ -123,12 +123,12 @@ static struct gzip_index *addpoint(struct gzip_index *index, uint8_t bits,
 
 
 /* Pretty much the same as from zran.c */
-int generate_index_fp(FILE* in, offset_t span, struct gzip_index** idx)
+int generate_index_fp(FILE* in, offset_t span, struct gzip_compression_info** idx)
```

Comment:
I'm not entirely sure what this name means, but it doesn't look right given the renaming.

Reply:
Agreed